### PR TITLE
docs(design): include design guidelines for new icon creation

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

Every time someone creates a new icon, there's a question of "what are the guidelines to follow?"

## Changes

Adding guidelines

### Added

Guidelines for icon structure

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Comment by chris-at-jobber** [comment]
@seanhealy looking for your thoughts on if this follows the order/format we talked about

**Comment by rebecca-li** [comment]
> * Design within a 20px grid on a 24px artboard
> * Square and Rectangles generally have a 2px radius
> * Irregular shaped corners have a 1px radius
> * Circle icons have a 20px diameter
> * Interior corners are square (do not have radius)

That was me, sorry

**Comment by chris-at-jobber** [comment]
@rebecca-li updated and broke out into sections, thinking it can be a bit more readable this way: https://5e751f76e5ff0400094ecf56--jobber-atlantis.netlify.com/components/icon to review

**Comment by rebecca-li** [comment]
LGTM

File: packages/components/src/Icon/Icon.mdx [hunk 1]
```diff
@@ -25,6 +25,27 @@ Use icons as intended. Provided icons have a specific, meaning. To avoid
 confusing users, it’s essential that each image be used in accordance with its
 meaning and recommended usage.
 
+## Design Guidelines
+
+When contributing a new icon, follow these guidelines:
+
+### Size and Weight
+
+- Design within a 20px grid on a 24px artboard
```
Comment (seanhealy) [inline packages/components/src/Icon/Icon.mdx]: I don't quite follow this. Do you mean the icon should be within a 20px container within a 24px canvas, effectively meaning that there will be a 2px buffer around it?

File: packages/components/src/Icon/Icon.mdx [hunk 2]
```diff
@@ -25,6 +25,27 @@ Use icons as intended. Provided icons have a specific, meaning. To avoid
...
+- Circular icons should have a 20px diameter
+- Lines should have a 2px weight
+
+### Corners
+
+- Right-angle corners should have a 2px radius
+- Irregular corners should have a 1px radius
```
Comment (seanhealy) [inline packages/components/src/Icon/Icon.mdx]: Is this true for angles both over and under 90°?

File: packages/components/src/Icon/Icon.mdx (hunk 1 above)
Comment (chris-at-jobber) [reply packages/components/src/Icon/Icon.mdx]: Yep! I can make that more clear

File: packages/components/src/Icon/Icon.mdx (hunk 2 above)
Comment (chris-at-jobber) [reply packages/components/src/Icon/Icon.mdx]: Good clarification

**Comment by chris-at-jobber** [comment]
@seanhealy PTAL

File: packages/components/src/Icon/Icon.mdx [hunk 3]
```diff
@@ -25,6 +25,29 @@ Use icons as intended. Provided icons have a specific, meaning. To avoid
...
+- Icons should have a height and width of 20px container within a 24px artboard,
+  effectively providing a 2px padding on all sides
+- Circular icons should have a 20px diameter
+- Lines should have a 2px weight
+
+### Corners
+
+- 90° corners should have a 2px radius
+- Greater than 90° corners should have a 1px radius
+- 1–90° corners should have an 0.5px radius
```
Comment (seanhealy) [inline packages/components/src/Icon/Icon.mdx]:
```suggestion
- Less than 90° corners should have an 0.5px radius
```

**Pull request merged**